pandas/core/base.py

```diff
@@ -340,11 +340,15 @@ def _aggregate(self, arg, *args, **kwargs):
             def nested_renaming_depr(level=4):
                 # deprecation of nested renaming
                 # GH 15931
-                warnings.warn(
-                    ("using a dict with renaming "
-                     "is deprecated and will be removed in a future "
-                     "version"),
-                    FutureWarning, stacklevel=level)
+                msg = textwrap.dedent("""\
+                using a dict with renaming is deprecated and will be removed
+                in a future version.
+
+                For column-specific groupby renaming, use keyword aggregation
```

Comment:
did you update any tests with this change? (if not let's add one that checks for the message in the warning)